src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2086,6 +2087,7 @@ def aks_agentpool_add(cmd,      # pylint: disable=unused-argument,too-many-local
                       eviction_policy=CONST_SPOT_EVICTION_POLICY_DELETE,
                       spot_max_price=float('nan'),
                       labels=None,
+                      max_surge=None,
```

Comment:
Can you add a validation in please to ensure that the percentage is > 0 and < 100 -- or ideally if we have tighter bounds then use that pls